### ISSUE TITLE
something about Height

### ISSUE THREAD

**Issue by xiaoxiaozai** [SnapKit/SnapKit#203]
Dear  the developer of the SnapKit:
     Excuse me sir, i want to get some help ....

the scene like this:
let avatarView:UIImageView = UIImageView(image: UIImage(named: "yunwei"))
        meIDCardView.addSubview(avatarView)
        avatarView.snp_makeConstraints { (make) -> Void in
            make.centerY.equalTo(meIDCardView)
            make.left.equalTo(meIDCardView.snp_left).offset(screen_width*0.056)
            make.width.height.equalTo(meIDCardView.snp_height).offset(-40)
        }

i need to make Constraints 
   for the "make.width.height.equalTo(meIDCardView.snp_height).offset(-40)"
i want to do like 
avatarView's height = meIDCardView's height  / 3 
but i can't get the Height in meIDCardView and i don't know how to do it ..please help me ,thanks~

**Comment by robertjpayne** [comment]
@xiaoxiaozai Hard to follow but I believe you want to use the `equalTo(otherView.height).multipliedBy(0.33)` API?

**Comment by xiaoxiaozai** [comment]
@robertjpayne  thank you very much~
My English is not very good, your understanding is correct.
nice~
